src/tweenable.rs

```diff
@@ -538,6 +541,44 @@ impl<T> Tween<T> {
         self
     }
 
+    /// Enable running a one-shot system upon completion.
+    ///
+    /// If enabled, the tween will run a system via a provided [`SystemId`] when the
+    /// animation completes. This is similar to the [`with_completed()`],
+    /// but uses a system registered by [`register_system()`] instead of a callback.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_tweening::{lens::*, *};
+    /// # use bevy::{ecs::event::EventReader, math::Vec3};
+    /// # use std::time::Duration;
+    /// let test_system_system_id = world.register_system(test_system);
```

Comment:
I think the example won't build because `world` is undefined.

Reply:
Tbh I had no idea what I was doing here, I thought it was just a harmless comment. I feel like I'm being ignorant of something there. I suppose we can just add:

```
let world = World::new();
```